helpers/email/email.py

```diff
@@ -63,5 +63,23 @@ def event_cancellation_notification(
             event_reject_reason=event_reject_reason
         )
 
+    def admin_invite(self, user_email, user_name):
+        """
+        send email notification when an admin is added
+            :params
+                - user_email: the email of the user being added as an admin
+                - user_name: the name of the user being added as an admin
+        """
+        subject = 'Admin Invite'
```

Comment:
@daniellamarr Good work, kindly make the email subject descriptive.

Reply:
I also suppose that we are `adding` a user to be an admin rather than `inviting` them. @daniellamarr

Reply:
Thank you for the feedback, could you be of help on a more descriptive email subject?

Reply:
How about something in the lines of `Converge - You're now an admin` , `Role update.....`?. Something that briefly tells what's really happening.

Reply:
Alright, thank you👍

Reply:
Done